include/uv.h

```diff
@@ -1110,6 +1118,9 @@ struct uv_fs_s {
   void* ptr;
   const char* path;
   uv_stat_t statbuf;  /* Stores the result of uv_fs_stat() and uv_fs_fstat(). */
+  const uv_dir_t* dir; /* Stores the result of uv_fs_opendir() */
```

Comment:
can you group these in an struct perhaps? That way accessing them would seem a bit more pleasant, I think.